### PR TITLE
Create ITEMS.md

### PR DESCRIPTION
**Pull request by fewyn**
Initial draft of list of ITOM and Rare Items used by garbo

### CODE REVIEW

File: ITEMS.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+| packaged knock-off retro superhero cape | Back               | Configureable Back-Item                                             |
+| Protonic accelerator pack               | Back               | Bust Ghosts for buffs and profit?                                   |
+| vampyric cloake pattern                 | Back               | adventures, item drops, and +meat                                   |
+| Asdon Martin                            | Campground         | Meat/Item/Booze Drops                                               |
+| haunted doghouse                        | Campground         | Fetches things!                                                     |
+| packaged cold medicine cabinet          | Campground         | Two Words: Horse Paste.                                             |
+| packet of mushroom spores               | Campground         | Mushrooms for food and booze                                        |
+| packet of tall grass seeds              | Campground         | Pokemon... but not!                                                 |
```
Comment (horrible-little-slime) [inline ITEMS.md:28]: Drops fertilizer, which sells for meat

File: ITEMS.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+| portable Mayo Clinic                    | Campground         | Additional adventures from food and increases your cholesterol.     |
+| potted tea tree                         | Campground         | Drink your tea!                                                     |
+| Source terminal                         | Campground         | Digitize wanderers, meat drops, other buffs etc.                    |
+| Witchess Set                            | Campground         | Free Fights                                                         |
```
Comment (horrible-little-slime) [inline ITEMS.md:32]: big +weight buff

**Review by horrible-little-slime — COMMENTED** [review]
Most of these comments aren't actually helpful in retrospect. The first one is, though! Item Of The Month, not The Of.

**Comment by docrostov** [comment]
my attempt to fix the linting issue didn't work in any way shape or form but at least the draft should give a decent idea of what i was going for. i added some text to explain what the document is and split the list into three sub-lists -- the first being relatively cheap things you should purchase for yourself because they're worth it, the second being clan stash stuff, the third being bind-to-account IOTMs using the robort as an example.

seriously though what the fuck is wrong with linting ach

File: ITEMS.md [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+| lucky gold ring                   | Accessory | Piraterealm reward. Extra funfunds & Freddies; very high MPA accessory, much higher than you'd think |
+| belt of loathing                  | Accessory | +10 familiar weight; fantastic Barf accessory if you don't have the above.                           |
+| Stooper                           | Familiar  | Liver +1, ergo, quite a few extra turns each day                                                     |
+| Gingerbread Mob Hit               | Skill     | Insta-kill (1x a day)                                                                                |
+| training scroll: Shattering Punch | Skill     | Insta-kill (3x a day)                                                                                |
+| defective Game Grid token         | Item      | Gives a small but always-there meat buff that overlaps your embezzlers                               |
+| fishy pipe                        | Item      | Acquire from the old man by the sea for a Damp Old Boot (50 sand dollars)                            |
```
Comment (horrible-little-slime) [inline ITEMS.md:15]: Mention the purpose of it--we use it to get fishy to fight 1 extra embezzler with an evyfish egg

File: ITEMS.md [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+
+| Item                              | Type      | Notes                                                                                                |
+| --------------------------------- | --------- | ---------------------------------------------------------------------------------------------------- |
+| mafia pointer finger ring         | Accessory | Gain meat on critical hits. Bad for embezzlers, but generally worn (almost) all day in Barf.         |
+| mafia thumb ring                  | Accessory | Gives an adventure on turn-taking combats 4% of the time. One of the best accessories.               |
```
Comment (horrible-little-slime) [inline ITEMS.md:8]: cheeng's spectacles are better

File: ITEMS.md [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+| Build-a-City Gingerbread kit            | Zone               | Zone with various items/buffs                                                      |
+| Chateau Mantegna room key               | Zone               | Fax/Copier, better resting, access to other items                                  |
+| Distant Woods Getaway Brochure          | Zone               | Free buffs                                                                         |
+| heart-shaped crate                      | Zone               | Zone where can gain various items, ex LOV Enamorang                                |
+| Horsery contract                        | Zone               | Horses with buffs!                                                                 |
+| shrine to the Barrel god                | Zone               | Drops barrels during combat, can pray for items/buffs                              |
+| The Neverending Party                   | Zone               | Various buffs                                                                      |
```
Comment (horrible-little-slime) [inline ITEMS.md:145]: Free fights is the primary utility here

File: ITEMS.md [hunk 6]
```diff
@@ -0,0 +1,147 @@
...
+
+So, to reiterate: please, _do not_ take this as a "suggested IOTMs to buy" list. This is simply a list of what Garbo supports. If you have an IOTM that is not included in this list, but you think it -should-, let us know in the #garbage-collector channel on the ASS Discord, and we can walk you through the process of submitting a pull request. Thank you for reading, and happy garbage farming!
+
+| Item                                    | Type               | Notes                                                                              |
+| --------------------------------------- | ------------------ | ---------------------------------------------------------------------------------- |
+| Beach Comb                              | Accessory          | Familiar weight beachcomb buff                                                     |
```
Comment (horrible-little-slime) [inline ITEMS.md:49]: also mention that COMBO is lit

File: ITEMS.md [hunk 7]
```diff
@@ -0,0 +1,147 @@
...
+| Guzzlr tablet                           | Accessory          | Garbo uses Guzzlr to extract a bit of value from wanderers, by gaining guzzlrbucks |
+| Kremlin's Greatest Briefcase            | Accessory          | Banish, buffs, & martinis                                                          |
+| Lil' Doctor™ bag                        | Accessory          | Banish and Insta-kill                                                              |
+| mayfly bait necklace                    | Accessory          | Used to snag a few more funfunds                                                   |
+| mint-in-box Powerful Glove              | Accessory          | Monster Replacer                                                                   |
```
Comment (horrible-little-slime) [inline ITEMS.md:54]: mention extrovermectin

File: ITEMS.md [hunk 8]
```diff
@@ -0,0 +1,147 @@
...
+| mafia thumb ring                  | Accessory | Gives an adventure on turn-taking combats 4% of the time. One of the best accessories.               |
+| lucky gold ring                   | Accessory | Piraterealm reward. Extra funfunds & Freddies; very high MPA accessory, much higher than you'd think |
+| belt of loathing                  | Accessory | +10 familiar weight; fantastic Barf accessory if you don't have the above.                           |
+| Stooper                           | Familiar  | Liver +1, ergo, quite a few extra turns each day                                                     |
+| Gingerbread Mob Hit               | Skill     | Insta-kill (1x a day)                                                                                |
+| training scroll: Shattering Punch | Skill     | Insta-kill (3x a day)                                                                                |
+| defective Game Grid token         | Item      | Gives a small but always-there meat buff that overlaps your embezzlers                               |
```
Comment (horrible-little-slime) [inline ITEMS.md:14]: move to stash items

File: ITEMS.md [hunk 9]
```diff
@@ -0,0 +1,147 @@
...
+| Grimstone Golem                         | Familiar           | Drops Item - grimstone mask (1)                                                    |
+| Li'l Xenomorph                          | Familiar           | Drops Item - transporter transponder (5)                                           |
+| Llama Lama                              | Familiar           | Drops Item - llama lama gong                                                       |
+| Machine Elf                             | Familiar           | Zone unlock with free fights.                                                      |
+| Mini-Hipster                            | Familiar           | Scaling Monster/Free Fights                                                        |
+| Ms. Puck Man                            | Familiar           | Drops Item - yellow pixel (25) & Power pill                                        |
+| Obtuse Angel                            | Familiar           | Drops Time arrow's for creating wanderers                                          |
```
Comment (horrible-little-slime) [inline ITEMS.md:90]: times arrow does not create wanderers

File: ITEMS.md [hunk 10]
```diff
@@ -0,0 +1,147 @@
...
+| --------------------------------- | --------- | ---------------------------------------------------------------------------------------------------- |
+| mafia pointer finger ring         | Accessory | Gain meat on critical hits. Bad for embezzlers, but generally worn (almost) all day in Barf.         |
+| mafia thumb ring                  | Accessory | Gives an adventure on turn-taking combats 4% of the time. One of the best accessories.               |
+| lucky gold ring                   | Accessory | Piraterealm reward. Extra funfunds & Freddies; very high MPA accessory, much higher than you'd think |
+| belt of loathing                  | Accessory | +10 familiar weight; fantastic Barf accessory if you don't have the above.                           |
+| Stooper                           | Familiar  | Liver +1, ergo, quite a few extra turns each day                                                     |
+| Gingerbread Mob Hit               | Skill     | Insta-kill (1x a day)                                                                                |
```
Comment (s-k-z) [inline ITEMS.md]: I think insta-kill should be renamed to free kill. Since insta-kills and free kills are distinct concepts.

File: ITEMS.md [hunk 11]
```diff
@@ -0,0 +1,147 @@
...
+
+| Item                                    | Type               | Notes                                                                              |
+| --------------------------------------- | ------------------ | ---------------------------------------------------------------------------------- |
+| Beach Comb                              | Accessory          | Familiar weight beachcomb buff                                                     |
+| Guzzlr tablet                           | Accessory          | Garbo uses Guzzlr to extract a bit of value from wanderers, by gaining guzzlrbucks |
+| Kremlin's Greatest Briefcase            | Accessory          | Banish, buffs, & martinis                                                          |
+| Lil' Doctor™ bag                        | Accessory          | Banish and Insta-kill                                                              |
```
Comment (s-k-z) [inline ITEMS.md:60]: 3/day

File: ITEMS.md [hunk 12]
```diff
@@ -0,0 +1,147 @@
...
+| pantogram                               | Item               | Sacrifice for pants with specific enchantments.                                    |
+| SongBoom™ BoomBox                       | Item               | Increase Meat Drops                                                                |
+| stinky cheese ball                      | Item               | Free fight, meat/item drops                                                        |
+| Time-Spinner                            | Item               | Copier                                                                             |
+| Undrilled cosmic bowling ball           | Item               | Item/Meat Drops, banishes                                                          |
+| cursed magnifying glass                 | Off-hand           | Free Fights                                                                        |
+| Kramco Sausage-o-Matic™                 | Off-hand           | Item Drops, Increase Crits, and make sausages                                      |
```
Comment (s-k-z) [inline ITEMS.md:129]: Also free fights.

File: ITEMS.md [hunk 13]
```diff
@@ -0,0 +1,147 @@
...
+| latte lovers member's mug               | Off-hand           | Fill your coffee mug for various buffs.                                            |
+| Cargo Cultist Shorts                    | Pants              | The pocketseses                                                                    |
```
Comment (s-k-z) [inline ITEMS.md:131]: Grab a valuable item 1/day.

File: ITEMS.md [hunk 14]
```diff
@@ -0,0 +1,147 @@
...
+| Airplane charter: That 70s Volcano      | Zone               | Volcano Mining and used for Guzzlr/Fire Extinguisher                               |
+| Airplane charter: The Glaciest          | Zone               | Used for Guzzlr/Fire Extinguisher                                                  |
+| Bastille Battalion control rig          | Zone               | Daily Buff                                                                         |
+| Boxing Day care package                 | Zone               | Free items, buffs, misc                                                            |
+| Build-a-City Gingerbread kit            | Zone               | Zone with various items/buffs                                                      |
+| Chateau Mantegna room key               | Zone               | Fax/Copier, better resting, access to other items                                  |
+| Distant Woods Getaway Brochure          | Zone               | Free buffs                                                                         |
```
Comment (s-k-z) [inline ITEMS.md:149]: Also better resting.

File: ITEMS.md [hunk 15]
```diff
@@ -0,0 +1,147 @@
...
+| SongBoom™ BoomBox                       | Item               | Increase Meat Drops                                                                |
+| stinky cheese ball                      | Item               | Free fight, meat/item drops                                                        |
+| Time-Spinner                            | Item               | Copier                                                                             |
+| Undrilled cosmic bowling ball           | Item               | Item/Meat Drops, banishes                                                          |
+| cursed magnifying glass                 | Off-hand           | Free Fights                                                                        |
+| Kramco Sausage-o-Matic™                 | Off-hand           | Item Drops, Increase Crits, and make sausages                                      |
+| latte lovers member's mug               | Off-hand           | Fill your coffee mug for various buffs.                                            |
```
Comment (s-k-z) [inline ITEMS.md:130]: Enchantments, not buffs.

File: ITEMS.md [hunk 16]
```diff
@@ -0,0 +1,147 @@
...
+| bottle of lovebug pheromones            | Item               | Summon lovebugs for meat and other misc effects.                                   |
+| can of Rain-Doh                         | Item               | Copier                                                                             |
+| Deck of Every Card                      | Item               | Draw cards for buffs, etc.                                                         |
+| January's Garbage Tote                  | Item               | Limited charge items for meat drops and other items.                               |
+| mime army shotglass                     | Item               | Your first 1-drunkenness booze item of the day doesn't count                       |
+| Monster Manuel                          | Item               | Factoids and knowing monster values.                                               |
+| mumming trunk                           | Item               | Dress up your familiars                                                            |
```
Comment (s-k-z) [inline ITEMS.md:122]: +Item and +Meat drop.

File: ITEMS.md [hunk 17]
```diff
@@ -0,0 +1,147 @@
...
+| vampyric cloake pattern                 | Back               | adventures, item drops, and +meat                                                  |
+| Asdon Martin                            | Campground         | Meat/Item/Booze Drops                                                              |
+| haunted doghouse                        | Campground         | Fetches things!                                                                    |
+| packaged cold medicine cabinet          | Campground         | Two Words: Horse Paste.                                                            |
+| packet of mushroom spores               | Campground         | Mushrooms for food and booze                                                       |
+| packet of tall grass seeds              | Campground         | Drops fertilizer, which sell for decent meat gain                                  |
+| portable Mayo Clinic                    | Campground         | Additional adventures from food and increases your cholesterol.                    |
```
Comment (s-k-z) [inline ITEMS.md:73]: Additional adventures or double effect duration from food.

File: ITEMS.md [hunk 18]
```diff
@@ -0,0 +1,147 @@
...
+| packaged knock-off retro superhero cape | Back               | Configureable Back-Item                                                            |
+| Protonic accelerator pack               | Back               | Bust Ghosts for buffs and profit                                                   |
+| vampyric cloake pattern                 | Back               | adventures, item drops, and +meat                                                  |
+| Asdon Martin                            | Campground         | Meat/Item/Booze Drops                                                              |
+| haunted doghouse                        | Campground         | Fetches things!                                                                    |
+| packaged cold medicine cabinet          | Campground         | Two Words: Horse Paste.                                                            |
+| packet of mushroom spores               | Campground         | Mushrooms for food and booze                                                       |
```
Comment (s-k-z) [inline ITEMS.md:71]: Free fights.

File: ITEMS.md [hunk 19]
```diff
@@ -0,0 +1,147 @@
...
+| Lil' Doctor™ bag                        | Accessory          | Banish and Insta-kill                                                              |
+| mayfly bait necklace                    | Accessory          | Used to snag a few more funfunds                                                   |
+| mint-in-box Powerful Glove              | Accessory          | Monster Replacer                                                                   |
+| packaged backup camera                  | Accessory          | Copier, +Meat Drops                                                                |
+| Unopened Eight Days a Week Pill Keeper  | Accessory          | Used to double the next potion.                                                    |
+| packaged knock-off retro superhero cape | Back               | Configureable Back-Item                                                            |
+| Protonic accelerator pack               | Back               | Bust Ghosts for buffs and profit                                                   |
```
Comment (s-k-z) [inline ITEMS.md:66]: Free fights, not buffs.

File: ITEMS.md [hunk 20]
```diff
@@ -0,0 +1,147 @@
...
+| Kremlin's Greatest Briefcase            | Accessory          | Banish, buffs, & martinis                                                          |
+| Lil' Doctor™ bag                        | Accessory          | Banish and Insta-kill                                                              |
+| mayfly bait necklace                    | Accessory          | Used to snag a few more funfunds                                                   |
+| mint-in-box Powerful Glove              | Accessory          | Monster Replacer                                                                   |
+| packaged backup camera                  | Accessory          | Copier, +Meat Drops                                                                |
+| Unopened Eight Days a Week Pill Keeper  | Accessory          | Used to double the next potion.                                                    |
+| packaged knock-off retro superhero cape | Back               | Configureable Back-Item                                                            |
```
Comment (s-k-z) [inline ITEMS.md:65]: guaranteed crit source for the pointer ring

File: ITEMS.md [hunk 21]
```diff
@@ -0,0 +1,147 @@
...
+
+| Item                                    | Type               | Notes                                                                              |
+| --------------------------------------- | ------------------ | ---------------------------------------------------------------------------------- |
+| Beach Comb                              | Accessory          | Familiar weight beachcomb buff                                                     |
+| Guzzlr tablet                           | Accessory          | Garbo uses Guzzlr to extract a bit of value from wanderers, by gaining guzzlrbucks |
+| Kremlin's Greatest Briefcase            | Accessory          | Banish, buffs, & martinis                                                          |
+| Lil' Doctor™ bag                        | Accessory          | Banish and Insta-kill                                                              |
+| mayfly bait necklace                    | Accessory          | Used to snag a few more funfunds                                                   |
```
Comment (s-k-z) [inline ITEMS.md:61]: Free runs from Basic Elemental.

File: ITEMS.md [hunk 22]
```diff
@@ -0,0 +1,147 @@
...
+| Comprehensive Cartography               | Skill              | Sometimes buffs after combat, fight specific monsters                              |
+| emotion chip                            | Skill              | various skills, item drops/etc                                                     |
+| Pocket Meteor Guide                     | Skill              | Combat skills, occasionally drops items or increases meat or items.                |
+| Sweet Synthesis                         | Skill              | Use spleen to increase meat drops/etc                                              |
+| Tome of Clip Art                        | Skill              | Borrowed time, banishes, etc                                                       |
+| Fourth of May Cosplay Saber             | Weapon             | Copy/Banish/Yellow Ray, also configurable                                          |
+| Industrial fire extinguisher            | Weapon             | Various Combat Skills                                                              |
```
Comment (s-k-z) [inline ITEMS.md:139]: Guaranteed Pickpockets.

**Review by s-k-z — CHANGES_REQUESTED** [review]

File: ITEMS.md (hunk 11 above)
Comment (docrostov) [reply ITEMS.md:60]: addressed in new pull

File: ITEMS.md (hunk 21 above)
Comment (docrostov) [reply ITEMS.md]: resolved in new push

File: ITEMS.md (hunk 20 above)
Comment (docrostov) [reply ITEMS.md]: resolved in new PR

File: ITEMS.md (hunk 19 above)
Comment (docrostov) [reply ITEMS.md]: resolved in new PR

File: ITEMS.md (hunk 18 above)
Comment (docrostov) [reply ITEMS.md]: resolved in new PR

File: ITEMS.md (hunk 17 above)
Comment (docrostov) [reply ITEMS.md]: resolved in new PR

File: ITEMS.md (hunk 16 above)
Comment (docrostov) [reply ITEMS.md]: resolved in new PR

File: ITEMS.md (hunk 12 above)
Comment (docrostov) [reply ITEMS.md]: resolved in new PR

File: ITEMS.md (hunk 15 above)
Comment (docrostov) [reply ITEMS.md]: resolved in new PR

File: ITEMS.md (hunk 13 above)
Comment (docrostov) [reply ITEMS.md]: resolved in new PR

File: ITEMS.md (hunk 22 above)
Comment (docrostov) [reply ITEMS.md]: resolved in new PR

File: ITEMS.md (hunk 14 above)
Comment (docrostov) [reply ITEMS.md]: resolved in new PR

**Review by docrostov — APPROVED** [review]

**Review by s-k-z — APPROVED** [review]

**Pull request merged**